### ISSUE TITLE
a Demo / Dummy site

### ISSUE THREAD

**Issue by xuv** [lisezmoi/lightbulb#3]
Is there a demo / dummy site where we can see it in action?

**Comment by raphaelbastide** [comment]
Yes, [doggifpage](http://doggifpage.com/) and [catgifpage](http://catgifpage.com) were made with Lightbulb.

**Comment by xuv** [comment]
OMG :) This is great. You've just invented this lightbulb thing is the thumblr-of-the-poor. Love it.

**Comment by raphaelbastide** [comment]
“thumblr-of-the-poor” ah ah, I guess it is!
